### PR TITLE
Fixed ms argument name

### PR DESCRIPTION
**Pull request by omichel**
Addresses #67.

### CODE REVIEW

**Comment by omichel** [comment]
:

**Comment by omichel** [comment]
:+1:

File: reference/connector.md [hunk 1]
```diff
@@ -214,9 +214,8 @@ int wb_connector_get_presence(WbDeviceTag tag);
 **Description**
 
 The `wb_connector_enable_presence()` function starts querying the
-[Connector](#connector)'s *presence* (see definition below) state each `ms`
-milliseconds.
-The provided `ms` argument specifies the [Connector](#connector)'s sampling period.
+[Connector](#connector)'s *presence* (see definition below).
+The provided `sampling_period` argument specifies the sampling period of the [Connector](#connector) and is expressed in milliseconds.
```
Comment (stefaniapedrazzi) [inline reference/connector.md]: In this case it seems that some information are lost during the rephrasing.
I would explicitly write that the sampling period refers to the rate at which the presence of the connector is queried.

File: reference/differentialwheels.md [hunk 2]
```diff
@@ -197,6 +197,7 @@ int wb_differential_wheels_get_encoders_sampling_period(WbDeviceTag tag);
 
 These functions allow the user to enable or disable the incremental wheel
 encoders for both wheels of the [DifferentialWheels](#differentialwheels) robot.
+The `sampling_period` argument defined the sampling period of the encoder and is expressed in milliseconds.
```
Comment (stefaniapedrazzi) [inline reference/differentialwheels.md]: typo: defined ->defines

File: reference/java-api.md [hunk 3]
```diff
@@ -669,30 +669,30 @@ The following tables describe the Java classes and their methods.
...
+| import com.cyberbotics.webots.controller.Servo;                                                                          |
+| public class [Servo](servo.md) extends [Device](#java_device) {                                                          |
+| &nbsp;&nbsp; public final static int ROTATIONAL, LINEAR;                                                                 |
+| &nbsp;&nbsp; public void [setPosition](servo.md#wb_servo_set_position)(double position);                                 |
+| &nbsp;&nbsp; public double [getTargetPosition](servo.md#wb_servo_set_position)();                                        |
+| &nbsp;&nbsp; public void [setVelocity](servo.md#wb_servo_set_position)(double vel);                                      |
+| &nbsp;&nbsp; public void [setAcceleration](servo.md#wb_servo_set_position)(double force);                                |
+| &nbsp;&nbsp; public void [setMotorForce](servo.md#wb_servo_set_position)(double motor\_force);                           |
+| &nbsp;&nbsp; public double [getMinPosition](servo.md#wb_servo_set_position)();                                           |
```
Comment (stefaniapedrazzi) [inline reference/java-api.md]: why did you change the order of the functions?
I would keep `getMiniPosition` and `getMaxPosition` together.

File: reference/motion.md [hunk 4]
```diff
@@ -149,11 +149,11 @@ motion file in milliseconds.
 The `wbu_motion_get_time()` function returns the current playback position in
 milliseconds.
 
-The `wbu_motion_set_time()` function allows to change the playback position.
-This enables, for example, to skip forward or backward. Note that, the position
+The `wbu_motion_set_time()` function allows to change the playback position in time.
+This allows the user to skip forward or backwards. Note that, the position time
```
Comment (stefaniapedrazzi) [inline reference/motion.md]: I would also change ``forward`` in ``forwards`` to be consistent with ``backwards``.

File: reference/motion.md [hunk 5]
```diff
@@ -149,11 +149,11 @@ motion file in milliseconds.
...
 can be changed whether the motion is playing or stopped. The minimum value is 0
 (beginning of the motion), and the maximum value is the value returned by the
-`wbu_motion_get_duration()` function (end of the motion).
+`wbu_motion_get_duration()` function (end of the motion). It is expressed in milliseconds.
```
Comment (stefaniapedrazzi) [inline reference/motion.md]: Instead of ``It`` I would write ``The time`` to make the sentence more understandable.

File: reference/motion.md (hunk 4 above)
Comment (stefaniapedrazzi) [inline reference/motion.md]: I'm not sure what you mean with ``position time``.
Maybe ``time position`` sounds better.

File: reference/robot.md [hunk 6]
```diff
@@ -191,13 +191,13 @@ This function synchronizes the sensor and actuator data between Webots and the
...
 
-The `ms` parameter specifies the number of milliseconds that must be simulated
+The `duration` parameter specifies the amount of time, expressed in milliseconds, that must be simulated
 until the `wb_robot_step()` function returns. Note that this is not real time
 but virtual (simulation) time, so this is not like calling the system's
 `sleep()`. In fact the function may return immediately, however the important
-point is that when it returns `ms` milliseconds of simulation will have elapsed.
-In other words the physics will have run for `ms` milliseconds and hence the
-motors may have moved, the sensor values may have changed, etc. Note that `ms`
+point is that when it returns, the requested duration of simulation time will have elapsed.
```
Comment (stefaniapedrazzi) [inline reference/robot.md]: This is an already existing sentence but I'm not sure that the verb tenses are correct:
[..] when it returns [..] will have elapsed

is not better
[..] when it returns [..] is elapsed 

In case the following sentence has to be updated accordingly.

**Comment by omichel** [comment]
:+1:

File: reference/robot.md [hunk 7]
```diff
@@ -191,13 +191,12 @@ This function synchronizes the sensor and actuator data between Webots and the
...
-The `ms` parameter specifies the number of milliseconds that must be simulated
+The `duration` parameter specifies the amount of time, expressed in milliseconds, that must be simulated
 until the `wb_robot_step()` function returns. Note that this is not real time
 but virtual (simulation) time, so this is not like calling the system's
-`sleep()`. In fact the function may return immediately, however the important
-point is that when it returns `ms` milliseconds of simulation will have elapsed.
-In other words the physics will have run for `ms` milliseconds and hence the
-motors may have moved, the sensor values may have changed, etc. Note that `ms`
+`sleep()`. Depending on the complexity of the simulation and execution mode, the function may return quickly. When it returns, the requested duration of simulation time is elapsed.
+In other words the physics will have run for the specified duration and hence the
```
Comment (stefaniapedrazzi) [inline reference/robot.md]: I think that now the verb tense 'will have run' doesn't make sense any longer.
Maybe simply 'is run'  or 'runs' is better.

**Comment by omichel** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
:+1:
